Source/Details/ASBasicImageDownloader.mm

```diff
@@ -183,14 +183,18 @@ @interface NSURLRequest (ASBasicImageDownloader)
 @end
 
 @implementation NSURLRequest (ASBasicImageDownloader)
-static const char *kContextKey = NSStringFromClass(ASBasicImageDownloaderContext.class).UTF8String;
+
+static const char *ContextKey() {
+  return class_getName(ASBasicImageDownloaderContext.class);
+}
+
 - (void)setAsyncdisplaykit_context:(ASBasicImageDownloaderContext *)asyncdisplaykit_context
 {
-  objc_setAssociatedObject(self, kContextKey, asyncdisplaykit_context, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
+  objc_setAssociatedObject(self, ContextKey(), asyncdisplaykit_context, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
```

Comment:
Instead of using a `static const` let's use the selector:
```
objc_setAssociatedObject(self, @selector(asyncdisplaykit_context), asyncdisplaykit_context, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
```